src/components/organisms/AuthenticationModal/AuthenticationModal.tsx

```diff
@@ -8,29 +8,44 @@ interface PropsType {
   show: boolean;
   onHide: () => void;
   afterUserIsLoggedIn?: () => void;
+  showLogin?: boolean;
 }
 
 const AuthenticationModal: React.FunctionComponent<PropsType> = ({
   show,
   onHide,
   afterUserIsLoggedIn,
+  showLogin,
 }) => {
-  const [showRegisterForm, setShowRegisterForm] = useState(false);
-  const displayLoginForm = () => setShowRegisterForm(false);
-  const displayRegisterForm = () => setShowRegisterForm(true);
+  const [showRegisterForm, setShowRegisterForm] = useState(!showLogin);
+
+  const displayLoginForm = () => {
+    setShowRegisterForm(false);
+  };
+
+  const displayRegisterForm = () => {
+    setShowRegisterForm(true);
+  };
+
+  const closeAuthenticationModal = () => {
+    setShowRegisterForm(!showLogin);
```

Comment:
If `showLogin=false` then doing this will keep the `{!showRegisterForm && <LoginForm..` open. Not a fan of using boolean to control login/register. Would prefer if you removed the `showLogin` prop and had `showAuth: 'login' | 'register'`.